### PR TITLE
Add support for U+E003 links in Lean 3.49.0

### PR DESCRIPTION
**Pull request by eric-wieser**

### CODE REVIEW

File: print_docs.py [hunk 1]
```diff
@@ -347,7 +347,24 @@ def load_json():
...
+    builtins_page = site_root + "foundational_types.html"
+    # These correspond to the heading `id`s in `foundational_types.j2`. They are currently generated
+    # by mistletoe and are therefore ugly!
+    anchor_for = dict(
+      Sort="codesort-ucode",
+      Type="codetype-ucode",
+      pi="pi-types-codeπ-a--α-β-acode",
+      forall="pi-types-codeπ-a--α-β-acode",
+      implies="pi-types-codeπ-a--α-β-acode",
+    )
```
Comment (eric-wieser) [inline print_docs.py:363]: I would prefer to just cleanup the markdown, but that would create conflicts with #173

**Comment by eric-wieser** [comment]
#deploy

**Comment by eric-wieser** [comment]
#deploy

**Comment by gebner** [comment]
Works as advertised!

**Pull request merged**